### PR TITLE
Created a eslint base config and extend from this in each package

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard22**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "nuke": "yarn workspaces foreach run nuke && npx rimraf node_modules",
     "typecheck": "yarn workspaces foreach run typecheck",
     "lint": "yarn workspaces foreach run lint",
+    "lint:fix": "yarn workspaces foreach run lint --fix",
```
Comment (VWSCoronaDashboard24) [inline package.json:14]: 🙌

**Review by VWSCoronaDashboard24 — APPROVED** [review]
Very nice improvement! 👍

File: eslint-overrides.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (VWSCoronaDashboard19) [inline eslint-overrides.json:1]: Why is there a separate file for the overrides? Can't we put these inside `eslint-config.json`?

Comment (VWSCoronaDashboard22) [reply eslint-overrides.json:1]: No, doesn't get picked up if we do it like that

**Review by VWSCoronaDashboard19 — APPROVED** [review]

**Pull request merged**